proposals/lip-0045.md

```diff
@@ -1157,7 +1071,7 @@ Let `genesisBlockAssetBytes` be the `data` bytes included in the block assets fo
 * Within each substore, check that all entries have a different `storeKey`.
 * For each entry in the chain data substore there must be a corresponding entry (with the same value of `storeKey`) in the outbox root, channel data, and chain validators substores and viceversa. However, if the chain account has status set to terminated, the entry in the outbox root must *not* be present.
 * For each entry in the terminated outbox substore there must be a corresponding entry (with the same value of `storeKey`) in the terminated state substore.
-* For each entry in the terminated state substore, if the property `initialized` is set to `false`, the corresponding entry in the terminated outbox substore must *not* be present. Furthermore, the `stateRoot` must be set to empty bytes and `mainchainStateRoot` to a 32-bytes value. Conversely, if the property `initialized` is set to `true`, the `stateRoot` must be set to a 32-bytes value and `mainchainStateRoot` to empty bytes.
+* For each entry in the terminated state substore, if the property `initialized` is set to `false`, the corresponding entry in the terminated outbox substore must *not* be present. Furthermore, the `stateRoot` must be set to empty bytes and `mainchainStateRoot` to a 32-bytes value. Conversely, if the property `initialized` is set to `True`, the `stateRoot` must be set to a 32-bytes value and `mainchainStateRoot` to empty bytes.
 * For each entry `chainValidators` in `interoperabilityAsset.chainValidatorsSubstore`, let `activeValidators = chainValidators.storeValue.activeValidators` and let `certificateThreshold = chainValidators.storeValue.certificateThreshold`:
   * `activeValidators` must have at least 1 element and at most `MAX_NUM_VALIDATORS` elements.
   * `activeValidators` must be ordered lexicographically by `blsKey` property.
```

Comment:
Regarding the following line below:

> Either `channel.messageFeeTokenID={chainID=1, localID=0}` (corresponding to the LSK token), or `channel.messageFeeTokenID.chainID` must be equal to `channelData.storeKey` or `ownChainAccount.ID` (the message fee token must be a native token of either chains).

Note that token IDs are now byte values and not objects so the notation should be adjusted according to the Token module.

Reply:
yes, fixed